Distribution/kubernetes/Magpie/magpie/app/backend/private/deployment.yaml

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: private-backend-deployment
+spec:
+  replicas: 1
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: private-backend
+  template:
+    metadata:
+      labels:
+        app: private-backend
+    spec:
+      containers:
+        - name: flood
+          image: ghcr.io/2024-cmpu9010-group-3/backend-private:0.2.1@sha256:c624444e50f1d012cc42e4870c8833913ee5446241143f3adeaa6684178f6075
+          resources: {}
+          ports:
+            - containerPort: 3000
+          env:
```

Comment:
If we have time, we should consider detecting if the backend is running in a container, right now it will spit out some warnings about the .env file missing because the env variable configuration is handled differently in K8s